code/__defines/misc.dm

```diff
@@ -74,8 +74,9 @@
 #define ASFX_DROPSOUND	16
 #define ASFX_ARCADE		32
 #define ASFX_RADIO		64
+#define ASFX_INSTRUMENT 128
```

Comment:
the spacings a little weird here

Reply:
![spacing](https://user-images.githubusercontent.com/47489928/88010310-83d45c00-cb0c-11ea-831e-231e3642434f.PNG)

Not sure what's going on there... I blame git? :S